notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -3186,6 +3186,63 @@ the scientist profession [Dr. Herbert West and Dr. Howard Phillips were experime
+torture [Kang was imprisoned in a workshop basement and subjected to numerous gruesome tortures.]
+coping with being tortured [Kang was imprisoned in a workshop basement and subjected to numerous gruesome tortures.]
+delusional point of view [The film is shown from the point of view of a deranged man, named Byeong-gu, who has kidnapped another man whom he is convinced is a space alien.]
+the making of a monster to society [The story explores how Byeong-gu's traumatic past led him to become a deranged and sadistic killer.]descent into madness [The story explores how Byeong-gu slowly went mad from all the violence that surrounded him.]
+aliens meddling in history [A revelation of the story is that aliens from Andromeda had created humans in their own image and had been guiding the development of humanity since prehistoric times.]
+sadism [Byeong-gu was exceeding sadistic in his treatment of his captive, Kang. Among other things, Byeong-gu delighted in crucifying Kang and crushing his shin with the back of an axe.]
+
+:: Minor Themes
+telepathic ability [Byeong-gu suppressed Kang's ability to communicate telepathically by shaving Kang's head.]
+master and pet [Byeong-gu fed the remains of Detective Choo to Byeong-gu' own dog, named Earth.]
+mother and son [Byeong-gu didn't want to let go of his comatose mother, and refused to pull the plug on her. Byeong-gu conversed with his ill mother in a flashback scene.]
+coping with the death of a parent [Byeong-gu was devastated when Kang tricked him into giving a lethal dose of benzene to Byeong-gu's own comatose mother.]
+human interpersonal relationship [Detective Kim was assigned to work on the Kang abduction case with together with Detective Choo, although Choo died before they had much of a chance to bond.]
+theft [A hospital worker reported the theft of $20,000 of restricted medicine to Detective Choo.]
+ransom kidnapping [The police at first assumed that Kang's captor was after a ransom.]
+drug abuse [Byeong-gu was addicted to methamphetamine and was shown taking blue meth pills here and there over the course of the film.]
+serial murder [Detective Kim uncovered that Byeong-gu had murdered three people that were currently listed as missing persons.]
+Atlantis [Kang explained that the Atlanteans perished in a nuclear holocaust and subsequent flood.]
+nuclear holocaust [Kang explained that the Atlanteans perished in a nuclear holocaust and subsequent flood.]
+theory of relativity [According to Kang, relativity theory explained how it could be that time passed more slowly on his homeworld as compared to Earth. The idea being that the more massive a planet is, the more slowly time passes on it.]
+flying saucer [Kang was picked up by a large alien saucer.]
+attack from outer space by a genocidal enemy [In the end, aliens from the Andromeda galaxy annihilated Earth, presumably extinguishing humanity in the act.]
```

Comment:
Given the other major themes it sound like this, or at least it's parent theme, ought to be major

Reply:
Reasonable. I suppose the "hero" of the story was making efforts to prevent the attack throughout. I knocked it up to major.